activities/codebase-stewardship/product-development.md

```diff
@@ -0,0 +1,117 @@
+|Product Backlog | A publicly visible shared product backlog which contains a description of categorized and prioritized features (epics and user stories) which can be related to containing procurable work packages | [Product Backlog](https://github.com/publiccodenet/product-steering/projects/1) part of [product-steering repository template](https://github.com/publiccodenet/product-steering)| Design(2), Realize(3), All phases |
+|Feature Request Template | An instruction inside the codebase explaining how to request features to be considered to be developed. The template SHOULD include a detailed description of a feature (user story) which supports developers and product stakeholders to be able to design, develop and procure this feature. | [Feature Request Template](https://github.com/publiccodenet/product-steering/blob/main/.github/ISSUE_TEMPLATE/feature_request.md) part of [product-steering repository template](https://github.com/publiccodenet/product-steering) | Design(2), Realize(3), Service(4), All phases |
+| Governance exercise | The governance exercise is designed to help civil servants develop a shared understanding and vocabulary of how to make open source projects successful in a government organization. | [LINK](https://about.publiccode.net/activities/supporting-codebase-governance/exercise/) | Design(2), Realize(3), Service(4), All phases|
+| Customized governance game | A customized governance workshop can be used for a codebase that already has some form of governance in place, or is considering a draft proposal. Can also be used to focus on a single aspect of governance for example, product management | [LINK](https://about.publiccode.net/activities/supporting-codebase-governance/customized-governance-workshop/)|Design(2), Realize(3), Service(4), All phases|
+| Workshop "Mapping business models enabled by a codebase" | Have a collaborative working session with new or existing community members to map out potential different business models that can be created around a given codebase. | [Link](https://about.publiccode.net/activities/workshops/mapping-business-models/) | Realize (3), Service (4), All phases
+| Workshop "Mapping risk and service level agreements workshop" | Have a collaborative working session with new or existing community members to map out, for a given codebase, potential different risks when using the codebase in production, as well as approaches to service level agreements. |[Link](https://about.publiccode.net/activities/workshops/mapping-risk-and-service-level-agreements/) | Service (4), All phases|
+
+## Further reading
```

Comment:
This section is really valuable for anyone who wants to dive deeper. But it would be even more valuable if every point had a sentence explaining it a bit more (like how the third point already has it).

Reply:
hi @Ainali could clarify what you mean with "like how the third party already has it?" Do I understand you correctly if you mean like an example in another codebase as a good example?

Reply:
No, the third point in the bullet point list in the further reading section. It is the only one with a sentence next to the link.